Dockerfile.sriov-network-config-daemon

```diff
@@ -5,7 +5,7 @@ RUN make _build-sriov-network-config-daemon BIN_PATH=build/_output/cmd
 
 FROM quay.io/centos/centos:stream9
 ARG MSTFLINT=mstflint
-RUN ARCH_DEP_PKGS=$(if [ "$(uname -m)" != "s390x" ]; then echo -n ${MSTFLINT} ; fi) && yum -y install hwdata $ARCH_DEP_PKGS && yum clean all
+RUN ARCH_DEP_PKGS=$(if [ "$(uname -m)" != "s390x" ]; then echo -n ${MSTFLINT} ; fi) && yum -y install hwdata pciutils $ARCH_DEP_PKGS && yum clean all
```

Comment:
I know that pciutils is needed because of mstflint. If it is not installed mstflint fails. Would there be a better way to handle this dependency. Otherwise looking at this you will never know there is such a dependency.

Reply:
We tried several things such as `chroot` into the `/root` from the host, but without any luck.
I can add a comment to the Dockerfile and the usage in `mellanox.go` that we need this package.

Reply:
+1 to add a comment